src/style/components/lexical-input.less

```diff
@@ -248,6 +251,19 @@
       list-style-type: lower-roman;
     }
   }
+
+  // keep the list indentation for the next paragraph after the list
+  & + .editor-paragraph,
+  // and for every paragraph with text with a parent with text
+  // the ltr class is present for a paragraph with left-to-right text
+  & ~ .ltr + .ltr {
+    margin-left: 24px;
+  }
```

Comment:
css spaghetti here.
lexical does not keep the next paragraph after a list in the ul, but markdownit does (it's standard md).

the css is for keeping the list indentation for a single paragraph after a list, and every paragraph after that that contain text (the ltr class is added by lexical to paragraphs with left-to-right text)